app/src/ActionModal/components/ModalDeposit.vue

```diff
@@ -126,7 +126,8 @@ export default {
         amount: {
           amount: this.amount,
           denom: this.denom,
-        }
+        },
+        depositsSum: this.deposits.length,
```

Comment:
this sounds like the sum of all deposits but is the count. Let's name `depositsCount`?

Reply:
what is this going to be used for?

Reply:
I think I explained this already like 4 times :sweat_smile: but maybe Fabo wasn't there :shrug: 

the `numberOfSeconds` that you happily deleted, @faboweb , is needed for the api.tx.democracy.second extrinsic. And it is the deposit count, exactly. So we had the `depositTx` broken due to you PR. I can change the name, but this needs to be here

Reply:
I see. I am sorry, I didn't see that. For clarification: The tx needs to provide a maximum count of existing seconds. So people don't second if there are already n people seconding. Strange UX...